### PR TITLE
Issue #13213: Removed //ok from onestatementperline

### PR DESCRIPTION
**Pull request by Tahanima**
Part of #13213

Removed `//ok` comments from the following

- InputOneStatementPerLineBeginTree2.java
- InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java
- InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd2.java
- InputOneStatementPerLineBeginTreeTest.java
- InputOneStatementPerLineTest.java
- InputOneStatementPerLineTryWithResources.java
- InputOneStatementPerLineTryWithResourcesIgnore.java

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 import java.io.*;
 
-// ok
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java:12]: Removal of this line changed the violation line.
You will need to change the violation lines of expected string array in the respective check.
Or you can simply add a blank line where the '// ok' comment has been removed.

**Review by MANISH-K-07 — COMMENTED** [review]
Please resolve conflicts....

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java (hunk 1 above)
Comment (Tahanima) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java:12]: Opted for the second solution.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
+
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java:12]: Please investigate the Pitest failures

**Review by MANISH-K-07 — COMMENTED** [review]
Pitest needs to be green in order for approval

**Comment by romani** [comment]
Red pitest is hard blocker for merge, other 2 are failing even in master, they are blockers.

**Pull request closed without merging**